File: Version 1.0/soaint-sgd-enterprise-artifacts/canonical-api-contract/src/main/java/co/com/soaint/foundation/canonical/correspondencia/ItemDevolucionDTO.java
package co.com.soaint.foundation.canonical.correspondencia;

import lombok.*;

import javax.xml.bind.annotation.XmlRootElement;
import java.io.Serializable;

/**
 * ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 * Soaint Generic Artifact
 * Created:03-Nov-2017
 * Author: esanchez
 * Type: JAVA class Artifact
 * Purpose: DTO - Model Artifact
 * ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 */

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
@XmlRootElement(namespace = "http://soaint.com/domain-artifacts/item-devolucion/1.0.0")
@ToString
public class ItemDevolucionDTO implements Serializable {

    private static final long serialVersionUID = 1L;
    AgenteDTO agente;
    Integer causalDevolucion;
    String funDevuelve;
    CorrespondenciaDTO correspondencia;
}

File: Version 1.0/soaint-sgd-ui/soaint-sgd-web/src/app/domain/itemDevolucionDTO.ts
import {AgentDTO} from './agentDTO';
import {CorrespondenciaDTO} from "./correspondenciaDTO";

export interface ItemDevolucionDTO {
  agente: AgentDTO;
  causalDevolucion: string;
  correspondencia: CorrespondenciaDTO;
  funDevuelve: String;


}
